pages/4_Documentation.py

```diff
@@ -323,6 +323,35 @@
          ( 'N.A.'           , 'The mutation site could not be predicted (for instance, because it is located in an unstructured region)') ]
 st.table(pd.DataFrame(data, columns=['Value', 'Meaning']))
 
+st.subheader("""Long range: AlloSigma2-PSN""")
+
+st.write("""This module is available only for ensemble mode and is designed to 
+validate the predictions of AlloSigma2 using a Protein Structure Network (PSN)-based method.
+
+The module relies on the output of the Long Range module, specifically the mutations predicted 
```

Comment:
I think you can be a bit more explicit - we use the same analysis we used for simple mode (so same model as well if we do, which is not really consistent with ensemble mode and so worth saying), that allows us to get response site that have certain features (filtering on allosteric ddg + pockets)